prow/README.md

```diff
@@ -155,3 +155,28 @@ $ chmod +x prow/my-presubmit.sh
 This repository (istio/test-infra) also provides Prow jobs.
 
 - `test-infra-presubmit` - run the linting and testing
+
+### Manually Trigger a Prow Job
+
+
+> **Never do this unless necessary AND you are authorized by istio EngProd team (istio.slack.com test-infra channel)**
+
+
+Connect to Prow cluster and then:
+```bash
+$ git clone https://github.com/kubernetes/test-infra
+$ bazel build //prow/cmd/mkpj   
+$ bazel-bin/prow/cmd/mkpj/mkpj --config-path ~/istio/test-infra/prow/config.yaml --job test-infra-presubmit | kubectl create -f -
+```
+And give the required information:
```

Comment:
Maybe say "for example" at the beginning to make it clear this is not additional steps the user has to follow